packages/ounit2-lwt/ounit2-lwt.2.2.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvaini+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ounit"
+dev-repo: "git+https://github.com/gildor478/ounit.git"
+bug-reports: "https://github.com/gildor478/ounit/issues"
+doc: "https://gildor478.github.io/ounit/"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {>= "1.11.0"}
+  "lwt"
+  "ounit" {= version}
```

Comment:
```suggestion
  "ounit2" {= version}
```
Are you sure this shouldn't be `ounit2`?

Reply:
Yes, you are right (it still works but as an indirect dependency).